docs/cep/aggregations/index.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 1
+title: Create Named Aggregations
+---
+
+A _named aggregation_ allows you to obtain aggregates in an incremental manner for a specified set of time periods.
+
+This not only allows you to calculate aggregations with varied time granularity, but also allows you to access them in an interactive  manner for reports, dashboards, and for further processing. Its schema is defined via the _aggregation definition_.
+
+You can also create [Distributed Aggregations](distributed-aggregations.md) or [Joins](join-aggregations.md).
+
+## Purpose
+
+A named aggregation allows you to retrieve the aggregate values for different time durations.
+That is, it allows you to obtain aggregates such as `sum`, `count`, `avg`, `min`, `max`, `count`, and `distinctCount`
+of stream attributes for durations such as `sec`, `min`, `hour`, and so on.
+
+This is of considerable importance in many analytics scenarios, because aggregate values are often needed for several time periods.
+This also ensures that the aggregations are not lost due to unexpected system failures, because aggregates can be stored in different persistence stores.
+
+## Syntax
+
+```sql
+CREATE AGGREGATION <aggregator name> WITH (store.type='<store type>', store.replication.type='<global or local'>, purge.enable='<true or false>', purge.interval='<purging interval>', purge.retention.period='<retention period>')
+select <attribute name>, <aggregate function>(<attribute name>) as <attribute name>, ...
+from <input stream>
+    group by <attribute name>
+    aggregate by <timestamp attribute> every <time periods> ;
+```
+
+## Parameters
+
+| Item    | Description |
+|---------------|---------|
+| store                      | Used to refer to the data store where the calculated aggregate results are stored. This annotation is optional. When no annotation is provided, the data is stored in the `in-memory` store. |
```

Comment:
@StoyanTc, is this bit about in-memory store correct?

Reply:
@StoyanTc, please answer this before I merge.

Reply:
@diana-macrometa - Yes, that's correct. Every aggregation keeps the actual data in memory.